mobile/android/logcat-helper.ts

```diff
@@ -16,17 +18,14 @@ export class LogcatHelper implements Mobile.ILogcatHelper {
 		if (deviceIdentifier && !this.mapDeviceToLoggingStarted[deviceIdentifier]) {
 			const adb: Mobile.IDeviceAndroidDebugBridge = this.$injector.resolve(DeviceAndroidDebugBridge, { identifier: deviceIdentifier });
 
-			// remove cached logs:
-			await adb.executeCommand(["logcat", "-c"]);
+			this.adbLogcat = await adb.executeCommand(["logcat"], { returnChildProcess: true });
```

Comment:
In case you have more than one device, `$logcatHelper` will be called for each of them and you'll cache the `adb logcat` instance of the last device. So after that you'll kill it only, restart only this instance, etc. Is this expected?

Reply:
didn't think about that, thank you. will fix